NEWS

```diff
@@ -11,6 +11,211 @@
 ### Features:
 ### Bugfixes:
 
+## 1.18.0-rc1 (November 21, 2024)
+### Feature
+#### UCP
+ * Improved log with explicit prompt for address v2 configuration on failure
+ * Detected device inter-node capability with a new attribute flag
+ * Enabled using CUDA staging buffers for pipeline protocols by default
+ * Added endpoint reconfiguration support for non-reused p2p scenarios
+ * Enabled non-cacheable memory domains, activated for GDR
+ * Improved message upon wireup failure
+ * Added new protocol infrastructure to store aggregated factors for all ranges
+ * Added user_data parameter to ucp_ep_query
+ * Added support for host memory pipeline through CUDA buffers for rendezvous protocol
+ * Added global VA infrastructure and memory region in absence of error handling
+ * Improved performance nodes naming to be more informative
+ * Enforced always running on the same thread in single thread mode
+ * Used registration cache overhead from environment variable
+ * Improved protocol selection by aggregating multiple performance segments
+ * Added UCP_MEM_MAP_BLOCK API flag to enforce blocking memory mapping
+ * Allowed up-to 64 lanes for systems with many transports or devices
+ * Added rendezvous performance variants
+ * Added usage tracker to worker
+ * Added support for different variants of rendezvous protocol
+ * Multiple protocol selection algorithm improvements
+#### UCT
+ * Improved API documentation for memory domain flags
+#### RDMA CORE (IB, ROCE, etc.)
+ * Enabled support for a broader range or DC initiator
+ * Added DC dcs_hybrid policy
+ * Reduced MLX5/DV stack size consumption
+ * Improved MLX5 log level when allocations are unsupported, for both WC and NC
+ * Added ODP support for verbs and mlx5dv
+ * Added environment variable to enable setting DC initiator capacity
+ * Enabled DDP support on mlx5dv
+ * Added support of cuda-managed memory on IB when ODP is available
+ * Enabled use of implicit ODP with relaxed ordering
+ * Improved GPUDirect detection support on IB
+ * Increased DC initiator default count to 32 for performance optimization
+ * Added ConnectX-8 device support to IB
+ * Improved logs by suggesting to increase PF_LOG_BAR_SIZE when UAR allocation fails
+ * Added support for subnet filter list on RoCE GID
+ * Added a reason when a resource's lane is unreachable
+ * Added IB MLX5 as a separate UCX modules with RPM
+ * Added initial GGA support
+ * Added setting IB DEVX atomic_mode based on device capabilities
+ * Removed DC keepalive
+ * Implemented user-mode registration approach for XGVMI
```

Comment:
not clear what it  is. @iyastreb, can you pls comment?

Reply:
https://github.com/openucx/ucx/pull/9745

Reply:
fixed